doc/netplan.md

```diff
@@ -190,6 +190,11 @@ Virtual devices
     device's MAC address as a unique identifier rather than a RFC4361-compliant
     Client ID. This has no effect when NetworkManager is used as a renderer.
 
+ ``dhcp-options`` (mapping)
```

Comment:
I really think this needs to be called 'dhcp-overrides', not 'dhcp-options'.  'dhcp-options' is bound to cause confusion with "options" as used in RFC2131.